UnityProject/Assets/Scripts/UI/Systems/Lobby/CharacterCustomization.cs

```diff
@@ -119,6 +120,8 @@ public class CharacterCustomization : MonoBehaviour
 		private CharacterSettings lastSettings;
 		private int currentCharacterIndex = 0;
 
+		private TextInfo TI = new CultureInfo("en-US", false).TextInfo;
```

Comment:
```suggestion
private TextInfo textInfo = CultureInfo.CurrentCulture.TextInfo;
```
Does this work?

Reply:
Just had to change line 1172 to reflect that as well but looks good!

Reply:
wonderful, I'm merging after the tests are done.